.github/silabs-builds-thread.json

```diff
@@ -0,0 +1,16 @@
+{
+    "lighting-app": [
+        {
+            "boards": ["BRD4186C", "BRD4187C", "BRD2601B", "BRD2703A"],
```

Comment:
How come you have to split by family?

Reply:
Really just personal preference - doesn't change anything.

Easier to make sure all the boards were there.